sdk/servicebus/Azure.Messaging.ServiceBus/MigrationGuide-WindowsAzure-ServiceBus.md

```diff
@@ -0,0 +1,473 @@
+
+### BrokeredMessage changed to Message
+
+The object which used to represent a message has been changed from BrokeredMessage to ServiceBusMessage. Convenience methods like CompleteAsync, AbandonAsync, and DeadLetterAsync which were called on BrokeredMessage directly are not available on ServiceBusMessage object, but can instead be achieved with the help of ServiceBusReceiver and ServiceBusProcessor.
+
+### Client constructors
+
+#### Service Bus client
+
+While we continue to support connection strings when constructing a client, the main difference is when using Azure Active Directory. We now use the new [Azure.Identity](https://www.nuget.org/packages/Azure.Identity) library to share a single authentication solution between clients of different Azure services.
+
+Authenticate with Active Directory:
+
+```C# Snippet:ServiceBusAuthAAD
+// Create a ServiceBusClient that will authenticate through Active Directory
+string fullyQualifiedNamespace = "yournamespace.servicebus.windows.net";
+await using var client = new ServiceBusClient(fullyQualifiedNamespace, new DefaultAzureCredential());
+```
+
+Authenticate with connection string:
+
+```C# Snippet:ServiceBusAuthConnString
+// Create a ServiceBusClient that will authenticate using a connection string
+string connectionString = "<connection_string>";
+await using var client = new ServiceBusClient(connectionString);
+```
+
+#### Administration client
+
+The `ServiceBusAdministrationClient` replaces the `NamespaceManager` from `WindowsAzure.ServiceBus`. For example usage please see the sample for [CRUD operations using the `ServiceBusAdministrationClient`](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/servicebus/Azure.Messaging.ServiceBus/samples/Sample07_CrudOperations.md).
+
+Authenticate with Active Directory:
+
+```C# Snippet:ServiceBusAdministrationClientAAD
+// Create a ServiceBusAdministrationClient that will authenticate using default credentials
+string fullyQualifiedNamespace = "yournamespace.servicebus.windows.net";
+ServiceBusAdministrationClient client = new ServiceBusAdministrationClient(fullyQualifiedNamespace, new DefaultAzureCredential());
+```
+
+Authenticate with connection string:
+
+```C# Snippet:ServiceBusAdministrationClientConnectionString
+// Create a ServiceBusAdministrationClient that will authenticate using a connection string
+string connectionString = "<connection_string>";
+ServiceBusAdministrationClient client = new ServiceBusAdministrationClient(connectionString);
+```
+
+### Sending messages
+
+Previously, in `WindowsAzure.ServiceBus`, you could send messages either by using a `QueueClient` (or `TopicClient` if you are targeting a topic) or the `MessageSender`.
+
+While the `QueueClient` allowed you to send and receive messages over a queue, the `MessageSender` allowed you to send messages to both queues and topics.

```

Comment:
```suggestion
```

Not sure that we need this context.  I'd presume developers were already using the T0 type and know why they chose it.